models/ed/R/met2model.ED2.R

```diff
@@ -33,9 +33,11 @@
 met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, lst = 0, lat = NA, 
                           lon = NA, overwrite = FALSE, verbose = FALSE, ...) {
   overwrite <- as.logical(overwrite)
-  
+
+  # deprecated?  
   library(rhdf5)
```

Comment:
Once all references to functions from these packages are replaced with their pkg::func equivalent, yes the library calls can be dropped. But I don't think anyone's gone through ED2 wrapper in that much detail yet, so leave them for now.